### ISSUE TITLE
String localization

### ISSUE THREAD

**Issue by ukii** [mozilla/voice-web#723]
Since multiple versions of Common Voice are active world-wide, I will be working on localizing the strings we are using.

Package: https://www.npmjs.com/package/react-localization

**Comment by Gregoor** [comment]
Great to hear that you're looking into that. It's on our roadmap for early 2018 as well, @mikehenrty might be able to say more about it.

For an i18n library, I'd also like to throw [js-lingui](https://github.com/lingui/js-lingui) into the mix as its translations are type-checked and support plural.s

**Comment by Gregoor** [comment]
Last week we learned about the localization efforts inside of Mozilla, specifically [Pontoon](https://pontoon.mozilla.org/) and [Fluent](http://projectfluent.org/). We’re eager to leverage these tools as they offer contributors a simple and direct way to provide translations, without needing to be familiar with Git and GitHub. The Pontoon website linked above (hopefully) makes that point for me. It’ll certainly be a bit more of an effort to get these tools configured, compared to other translation libraries we talked about, but it also gives us more. I’m happy to take on the configuration efforts and after that find a way to bring in the work that’s already been done on localization.
As always, happy to hear your thoughts on this!

On a more technical note, here are some early TODOs I stumbled over, which I want to flag here so we don’t forget:
Fluent requires Node >= 8, so we either have to upgrade prod (which isn’t a bad idea anyways) or ask Stas if that number can be lowered
Use new version of UglifyJS (otherwise fluent didn’t work in prod for me): https://webpack.js.org/plugins/uglifyjs-webpack-plugin/#install
